scripts/update-ctags.py

```diff
@@ -13,12 +13,18 @@
 dstdir = os.path.abspath(sys.argv[2])
 
 os.chdir(dstdir + '/parsers')
-parser_dst_files = glob.glob('*.c') + glob.glob('*.h')
+parser_dst_files = glob.glob('**/*.c') + glob.glob('**/*.h')
```

Comment:
Maybe I would slightly prefer mirroring the ctags directory structure - i.e. having peg outside the parsers directory. That would mean explicitly mentioning `peg` and `parsers` directories in the globs and in the code below eliminating the
```
    if os.path.exists('parsers/' + f):
         shutil.copy('parsers/' + f, dstdir + '/parsers/' + f)
```
part.

Reply:
Ok, I can add `ctags/peg/` directory.